### ISSUE TITLE
Messaging endpoints

### ISSUE THREAD

**Issue by shubham0804** [etsy/open-api#67]
**Is your feature request related to a problem? Please describe.**
Need to integrate messages sent and received by sellers.

**Describe the solution you'd like**
Webhook to receive incoming Etsy messages  and a post request to send messages

**Describe alternatives you've considered**
None at the moment

**Additional context**

**Comment by FailMelon** [comment]
This is also a feature I'd be interested in, this is the only thing stopping me from having one of my listings completely automated through Etsy without having to resort to emails. Not only that as much as i like the Esty messaging system (its way better then eBays) but notifications suck and it'd be much nicer to have the ability to do Webhook's like op said. For the products i do on Etsy i basically have to stare at my store page right now for messages, this would stop me needing to do that.

**Comment by xpresas** [comment]
I completely agree. If I could get/send messages thru webhook or an agent. Would make our life easier...

**Comment by apexj** [comment]
I 100% agree.  Messages in the API would be extremely helpful with our system.

**Comment by viesoftware** [comment]
Actually hope Etsy Team will publish feature messaging on the next version.

**Comment by XAMelleOH** [comment]
Any feedback from Etsy team on this topic? 

FYI @etsy @Vaibhav @manjuraj

**Comment by xpresas** [comment]
Nah to be honest etsy moves really slowly with its features. We are still using the old API just because the new one is not worth the hassle to move.

**Comment by flmu** [comment]
Are there any updates on this topic?

FYI  @manjuraj
